build_ros1.sh

```diff
@@ -34,6 +34,7 @@ docker run -it --rm \
   -e PYTHON2_MAJOR_VERSION=${PYTHON2_MAJOR_VERSION} \
   -e PYTHON2_MINOR_VERSION=${PYTHON2_MINOR_VERSION} \
   -e PYTHON3_VERSION=${PYTHON3_VERSION} \
+	-e INSTALL_ROOT=${INSTALL_ROOT} \
```

Comment:
There's a mix of tabs and spaces in the changes, could you change it to always use spaces?